web/src/utils/date.ts

```diff
@@ -26,3 +33,18 @@ export const formatMonthAndDay = (month: number, day: number): string =>
   DateTime.fromObject({ month, day }).toFormat('d LLLL')
 
 export const suppressMilliInISO = (iso: string): string => iso.replace(/\.\d{0,3}/, '') // Using RegExp to remove the "." and milliseconds
+
+export const formatDateInUTC0 = (dtISO: string): string => {
```

Comment:
what's the significance of the `0` at the end of the function name?

Reply:
I want an ISO string that represents the date in terms of hours offset from UTC, in this case, with an offset of 0 hours.

I agree the function name is no good. Ideas?

Reply:
if it's an offset of 0 hours, wouldn't `formatDateInUTC` describe the functionality accurately?

Reply:
I wish. But no. You can omit the 00:00 part, and just say Z. Both are in UTC. I need the one with 00:00.

Reply:
`formatDateInUTC00Suffix`? I don't know. I can't think of an elegant name.

Reply:
I love that name.